handbook/sales/index.md

```diff
@@ -184,6 +188,10 @@ If a deal comes through a referral or introduction, tell [BizOps](../bizops/inde
 1. Update the ‘Closed Lost Dropdown’ property to reflect the reason. If the reason doesn’t exist in the dropdown, you can talk to [BizOps](../bizops/index.md) about adding one
 1. Expand upon the reason in the longform ‘Closed Lost Reason’ field. This supports [2019-Q4 OKR 1 v](../../company/goals/2019_q4.md) to identify the top 3 reasons potential customers don't sign.
 
+### When a customer churns
```

Comment:
Does this section apply if a contract size falls, or only if it goes to zero?

Reply:
Goes to zero for now